### PR TITLE
Adds the ability to record video, and improves the report a bug template

### PR DESCRIPTION
**Pull request by orta**
screen shot 2017-04-18 at 19 58 38

Talked to @katarinabatina  about a wishlist for reporting a bug. So now we have a bit longer delay on attaching images and it lets you record videos, and fixes the damn `Optional("Name")` issue that just became too much for me. Also mentions prod/staging and your signal strength.

screen shot 2017-04-18 at 19 59 23

### CODE REVIEW

**Comment by orta** [comment]
Updates the HockeySDK, because why not too.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2837,6 +2840,7 @@
 		49BA7E021655ABE600C06572 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				60D312461EA6CFD000B13E59 /* ReplayKit.framework */,
```
Comment (mennenia) [inline Artsy.xcodeproj/project.pbxproj]: R-r-r-rrreeewiiiiind!

File: Artsy/App/ARHockeyFeedbackDelegate.swift [hunk 2]
```diff
@@ -2,21 +2,30 @@ import UIKit
...
+
+            UIView.animate(withDuration: 0.15) {
+                ARTopMenuViewController.shared().view.alpha = 0.5
+            }
+
             // When I looked at how Hockey did this, I found that they would delay by a second
             // presumably it can take a second to have the image saved in the asset store before
-            // we can pull it out again. We might be able to get away with 0.5.
-            ar_dispatch_after(0.5, self.showFeedbackWithRecentScreenshot)
+            // we can pull it out again. We could not get away with 0.5, so 1.0.
```
Comment (mennenia) [inline Artsy/App/ARHockeyFeedbackDelegate.swift]: 👍

File: Artsy/App/ARHockeyFeedbackDelegate.swift [hunk 3]
```diff
@@ -27,28 +36,67 @@ class ARHockeyFeedbackDelegate: NSObject {
...
-        let user = User.current()
-        let initialMessage = NSString(string: "From: \(user?.name ?? "Onboarding user")\n\n")
+        var messages: [String] = []
+        let user = User.current().name ?? "Onboarding user"
+        messages.append("From: \(user)")
+
+        let staging = UserDefaults.standard.bool(forKey:ARUseStagingDefault)
+        messages.append(staging ? "On: Staging" : "On: Production")
+
+        messages.append("Signal: \(getSignalStrength())")
```
Comment (mennenia) [inline Artsy/App/ARHockeyFeedbackDelegate.swift]: nice additions!

File: Artsy/View_Controllers/Auction/AuctionViewController.swift [hunk 4]
```diff
@@ -287,8 +287,12 @@ extension AuctionViewController {
 
     /// Displays the current items, sorted/filtered through the current refine settings.
     func displayCurrentItems() {
-        let items = saleViewModel.refinedSaleArtworks(refineSettings)
-
+        var items = saleViewModel.refinedSaleArtworks(refineSettings)
+        var i = 0
+        items = items.filter({ (saleArtwork) -> Bool in
```
Comment (mennenia) [inline Artsy/View_Controllers/Auction/AuctionViewController.swift]: ?

Comment (orta) [inline Artsy/View_Controllers/Auction/AuctionViewController.swift]: good catch, was probably debugging something with Ash on my computer!

**Comment by mennenia** [comment]
woop woop! Mergin'

**Comment by orta** [comment]
Awesome beans

**Comment by ashfurrow** [comment]
Nice!

**Pull request merged**